code/game/objects/items/clown_items.dm

```diff
@@ -32,8 +32,11 @@
 
 /obj/item/soap/Initialize(mapload)
 	. = ..()
-	AddComponent(/datum/component/slippery, 80)
-	AddComponent(/datum/component/cleaner, cleanspeed, 0.1, pre_clean_callback=CALLBACK(src, PROC_REF(should_clean)), on_cleaned_callback=CALLBACK(src, PROC_REF(decreaseUses))) //less scaling for soapies
+	// SPLURT EDIT START - CYBORGS - their tongues handle cleaning by themselves
+	if(!istype(src, /obj/item/soap/dogborg_tongue))
+		AddComponent(/datum/component/slippery, 80)
+		AddComponent(/datum/component/cleaner, cleanspeed, 0.1, pre_clean_callback=CALLBACK(src, PROC_REF(should_clean)), on_cleaned_callback=CALLBACK(src, PROC_REF(decreaseUses))) //less scaling for soapies
+	// SPLURT EDIT END
```

Comment:
It might be possible to modularize this by just removing the component after its initialization in case it's an /obj/item/soap/dogborg_tongue